yaml/wp_energy_meter.yaml

```diff
@@ -0,0 +1,20 @@
+defaults:
+  internal: "false"
```

Comment:
I think you are right I just wanted to give the oportinity to check on the original numbers in case some numbers are mixed up or someone has already created frontends and does not have the time for breaking changes.

Reply:
Since we don't force the update on them, I think this kind of change should be fine. One can update whenever they have time